packages/itmat-interface/test/serverTests/_loginHelper.ts

```diff
@@ -13,12 +15,14 @@ export function connectUser(agent) {
     return connectAgent(agent, 'standardUser', 'admin');
 }
 
-export function connectAgent(agent, user, pw) {
+export function connectAgent(agent, user, pw, otpSecret = "H6BNKKO27DPLCATGEJAZNWQV4LWOTMRA") {    
+    const otp = mfa.generateTOTP(otpSecret);
```

Comment:
This probably doesn't belong here, My understanding is that it will generate a code that we expect to be good anyway. Furthermore, I do not see a test that verifies that the validation fails if no token is sent or an invalid one, this should be added to the test suite.

Reply:
Thank Florian for pointing it out. I've modified the code to clarify it better. The otpSecret for each users is randomly generated (except hardcoded for admin and standardUser at bootstrap).
I also add a new test case for the incorrect token :)